centre-de-recherche-sur-les-civilisations-de-l-asie-orientale.csl

```diff
@@ -361,7 +359,14 @@
   <macro name="issue">
     <choose>
       <if is-numeric="issue">
-        <text variable="issue" form="short" prefix=", n˚ "/>
+        <choose>
+          <if variable="volume">
+            <text variable="issue" form="short" prefix=", n˚ "/>
```

Comment:
Do not include a preceding comma or space in a prefix like this. Please move the comma to a delimiter on a group element.

Reply:
Very well